packages/stream-node/reader.js

```diff
@@ -0,0 +1,39 @@
+// @ts-check
+/// <reference types="ses"/>
+
+// This module provided for sake of fewer head scratches.
+// Node.js readable streams satisfy the signature of an async iterable iterator.
+// They however iterate Node.js Buffer values and are not hardened, so this
```

Comment:
Node streams iterate anything, their values may not be `Buffer` if they are in object mode. Also newer versions of Node actually have `Buffer` as a subclass of `UInt8Array`.